ci/test.py

```diff
@@ -138,7 +138,7 @@ def assertSubprocess(self, *args, capture=False, returncode=0, **kwargs):
     def _write_host_command(self, args, quiet_init=False):
         cmd_file = os.path.join(self.thread_dir, 'command')
         with open(cmd_file, 'w') as f:
-            f.write('. ./poky/pyrex-init-build-env%s && ' % ('', ' > /dev/null')[quiet_init])
+            f.write('. ./poky/pyrex-init-build-env%s && ' % (' > /dev/null 2>&1' if quiet_init else ''))
```

Comment:
> which causes errors in the test suite.

Does this just mean that we treat any output to stderr as a fatal error?

Reply:
No, there are tests where we parse stdout for output or to investigate state from inside the container. When the subprocess is invoked, there is an implicit "redirect stderr to stdout" as if the command were run like:

`(command) 2>&1`

The reason for this is to make the testing code easier and prevent deadlock when running.

Thus, before this change, the init command (when "quiet") was effectively:

`(. pyrex-init-build-env > /dev/null) 2>&1`

which meant that any stderr output would appear on stdout and get parsed by our tests, causing failures. With this change, the command becomes:

`(. pyrex-init-build-env > /dev/null 2>&1) 2>&1`

which correctly suppresses stderr when "quiet"

Reply:
I think that's what I meant. This is something we inflict on ourselves by not being prepared to handle any entries in stderr (which is fine).

Just wanted to make sure I understood.